examples/src/main/scala/eu/neverblink/jelly/examples/PekkoStreamsDecoderFlow.scala

```diff
@@ -167,7 +178,7 @@ object PekkoStreamsDecoderFlow extends shared.ScalaExample:
     val streamOptions = Await.result(snoopFuture._1, 10.seconds)
     val decodedQuads2 = Await.result(snoopFuture._2, 10.seconds)
 
-    val streamOptionsIndented = ("\n" + streamOptions.get.toProtoString.strip).replace("\n", "\n  ")
+    val streamOptionsIndented = ("\n" + streamOptions.get.toString.strip).replace("\n", "\n  ")
```

Comment:
This will not work – please run the example and see the output.

To not overcomplicate this, maybe instead select a few important fields from the stream options to be displayed and print them to console? Like stream type, table sizes.